mods/seasonal/moves.js

```diff
@@ -197,6 +197,31 @@ exports.BattleMovedex = {
 		target: "normal",
 		type: "Dark",
 	},
+	// HiMyNamesL
+	yallarestupid: {
+		accuracy: 100,
+		category: 'Status',
+		shortDesc: 'Confuse opponent, +1 spa, -1 opponent acc',
+		id: 'yallarestupid',
+		isNonstandard: true,
+		name: "Y'all are stupid!",
+		pp:10,
+		flags: {mirror: 1, protect: 1, reflectable: 1},
+		volatileStatus: 'Confusion',
```

Comment:
Typically they lowercase confusion

Reply:
good catch, needs to be lower case or it wont work iirc

Reply:
Most likely